src/components/sidebar-menu/SidebarMenu.tsx

```diff
@@ -15,25 +15,29 @@ import { styles } from '~/components/sidebar-menu/SidebarMenu.styles'
 interface SidebarMenu {
   tabsData: TutorProfileProps
   handleClick: (tab: TutorProfileTabsEnum) => void
+  activeTab: TutorProfileTabsEnum
 }
 
-const SidebarMenu: FC<SidebarMenu> = ({ handleClick, tabsData }) => {
+const SidebarMenu: FC<SidebarMenu> = ({ handleClick, tabsData, activeTab }) => {
   const { t } = useTranslation()
 
   const list = Object.keys(tabsData).map((key) => {
     const tabKey = key as TutorProfileTabsEnum
     const item = tabsData[tabKey]
+
+    const isActiveTab = item.title.endsWith(activeTab)
```

Comment:
Consider using some `activeTab` id or other identifier instead of checking tittle

Reply:
Discussed this solution on the meeting